helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java

```diff
@@ -42,12 +45,17 @@
 import org.apache.helix.model.IdealState.RebalanceMode;
 import org.apache.helix.model.Message;
 import org.apache.helix.tools.ClusterVerifiers.BestPossibleExternalViewVerifier;
+import org.apache.helix.tools.ClusterVerifiers.ClusterLiveNodesVerifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testng.Assert;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class TestPartitionMovementThrottle extends ZkStandAloneCMTestBase {
+  private static final Logger LOGGER = LoggerFactory.getLogger(TestPartitionMovementThrottle.class);
```

Comment:
Let's not log in the tests. If you need to debug, use print statements in your local environment.

And let us use LOG for the logger if you need to use it in other parts of the codebase.